### PR TITLE
Task/Implement find payments by user id.

### PR DESCRIPTION
**Pull request by MolchanovAlexander**

### CODE REVIEW

File: src/main/java/com/ua/accommodation/service/impl/StripeServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+            PaymentResponseDto responseDto = paymentMapper.toDto(payment);
+            responseDto.setMessage("Payment successful.");
+            return responseDto;
+        }
+        if (session.getPaymentStatus().equals("unpaid")) {
+            PaymentResponseDto responseDto = paymentMapper.toDto(payment);
+            responseDto.setMessage("Payment paused, you can complete it later.");
+            return responseDto;
+        }
+        return null;
```
Comment (TonyH277) [inline src/main/java/com/ua/accommodation/service/impl/StripeServiceImpl.java:94]:
```suggestion
            PaymentResponseDto responseDto = paymentMapper.toDto(payment);
            responseDto.setMessage("Payment paused, you can complete it later.");
            return responseDto;
```

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -7,8 +7,9 @@ spring.datasource.username=${POSTGRES_USER}
 spring.datasource.password=${POSTGRES_PASSWORD}
 
 stripe.secret.key=${STRIPE_SECRET_KEY}
+stripe.redirect.url=${STRIPE_REDIRECT_URL}
 
-telegram.bot.username=${TELEGRAM_BOT_NAME}
+telegram.bot.username=${TELEGRAM_BOT_USERNAME}
```
Comment (TonyH277) [inline src/main/resources/application.properties:12]: actually here we pass bot name, not username

File: src/main/resources/db/changelog/changes/create-booking-table.yaml [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
+      author: Molchanov O
       changes:
         - createTable:
             tableName: bookings
             columns:
               - column:
                   name: id
-                  type: BIGSERIAL
+                  type: BIGINT
+                  autoIncrement: true
```
Comment (TonyH277) [inline src/main/resources/db/changelog/changes/create-booking-table.yaml:12]: its better to create new chagneset and set yourself as an author

**Review by TonyH277 — CHANGES_REQUESTED** [review]
Pls create new changeset and make nessessary changes there, also we should have BOT_NAME not BOT_USER_NAME.

**Review by TonyH277 — APPROVED** [review]
Nice! pls rename pull name

**Review by VolandevlodD — APPROVED** [review]

**Pull request merged**